src/main/java/uk/gov/hmcts/reform/userprofileapi/config/SecurityEndpointFilter.java

```diff
@@ -21,9 +21,14 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
             throws ServletException, IOException {
 
         try {
+            log.debug("Inside Security EndPoint Filter" + request.getContextPath() + request.getPathInfo()
+                    + request.getRequestURI() + request.getQueryString() + request.getRequestedSessionId());
             filterChain.doFilter(request, response);
+            log.debug("After Security EndPoint Filter" + response);
         } catch (Exception e) {
             Throwable throwable = e.getCause();
+            log.error("caught an exception in filter"
+                    + throwable);
```

Comment:
this log will be printed in prod as well. Can you add separate log.debug for this?

Reply:
Thanks @SujeethKumarAri Addressed the above 👆  :)